### PR TITLE
Project name update app 30480

### PR DESCRIPTION
**Pull request by fin7710**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
-EASE plugin
+Arxan MAM Publisher
 -----------
 
-This plugin adds the ability to upload Android, Blackberry and iOS apps
+This plugin adds the ability to upload Android, iOS, and Windows apps
```
Comment (johntcaldas) [inline README.md:4]: 👍

File: pom.xml [hunk 2]
```diff
@@ -7,8 +7,11 @@
         <!-- which version of Jenkins is this plugin built against? -->
     </parent>
 
+    <!-- Unique ID for the plugin.  Do not change this. -->
     <artifactId>ease-plugin</artifactId>
-    <version>1.2.13-SNAPSHOT</version>
+    <!-- Name that displays on the 'Manage Plugins' page of Jenkins. -->
+    <name>Arxan MAM Publisher</name>
```
Comment (johntcaldas) [inline pom.xml:13]: I'm guessing this works because artifactId is never visible to the user?

File: src/main/resources/index.jelly [hunk 3]
```diff
@@ -3,7 +3,7 @@
   Description of a plugin in installed plugins page.
 -->
 <div>
-    This plugin adds the ability to upload Android, Blackberry and iOS apps.
+    This plugin adds the ability to upload Android, iOS, and Windows apps.
```
Comment (johntcaldas) [inline src/main/resources/index.jelly:6]: 👍 👍

**Review by johntcaldas — APPROVED** [review]
👍 👍 👍

File: pom.xml (hunk 2 above)
Comment (fin7710) [reply pom.xml:13]: It's displayed by default if you don't have a 'name' tag.  Adding the 'name' tag makes artifactId invisible to the user.

**Pull request merged**